api/app/signals/apps/api/v1/serializers/routing.py

```diff
@@ -0,0 +1,18 @@
+from rest_framework import serializers
+
+from signals.apps.api.generics.permissions import SIAPermissions
+from signals.apps.api.v1.serializers.nested import _NestedDepartmentModelSerializer
+from signals.apps.signals.models import RoutingExpression
+
+
+class RoutingDepartmentSerializer(serializers.ModelSerializer):
```

Comment:
I see (from ` api/app/signals/apps/api/v1/serializers/expression.py`) why you call it `RoutingDepartmentSerializer` but I would go for `RoutingExpressionSerializer` (closer to the existing naming convention).

Reply:
check